multiplayer/src/components/KeyboardControlsInfo.tsx

```diff
@@ -32,6 +34,14 @@ export default function Render() {
                     </div>
                 );
             })}
+            <div className="keymap-row tw-flex tw-flex-row tw-items-center">
+                <div className="keymap-key tw-align-middle tw-min-w-[2.5rem] tw-text-center tw-pb-[2.1rem]">
+                    1-6
```

Comment:
nit, either way is fine, but personally it'd have a slight preference for spaces around the dash (1 - 6).